src/utils/PatchError.js

```diff
@@ -52,8 +52,8 @@ export default class PatchError extends Error {
     for (let line = displayStartLine; line <= displayEndLine; line++) {
       let startOfLine = lineMap.indexForLocation({ line, column: 0 });
       let endOfLine = lineMap.indexForLocation({ line: line + 1, column: 0 });
-      if (isNaN(endOfLine)) {
-        if (isNaN(startOfLine)) {
+      if (endOfLine == null) {
```

Comment:
The docs for nullable types don't say anything about `undefined`, at least that I can find: https://flowtype.org/docs/five-simple-examples.html#2-adding-type-annotations.